tests/integration/src/test/resources/python/app_mock.py

```diff
@@ -90,6 +91,23 @@ def check_health(self):
             logging.debug("Done processing health request.")
             return
 
+        def handle_ipc_info(self):
+            logging.debug("Reporting IPC info")
+            df_shm_info = subprocess.check_output(["df", "-m", "/dev/shm"])
+            ipc_ns_info = subprocess.check_output(["stat", "-Lc", "%i", "/proc/self/ns/ipc"])
+
+            self.send_response(200)
+            self.send_header('Content-type', 'text/html')
+            self.end_headers()
+
+            self.wfile.write(df_shm_info)
```

Comment:
Maybe this should be JSON?

```
{ "df-shm": "{process output...}",
  "stat-ipc": "{...}"
}
```

Reply:
Or, perhaps better yet, why not just have to separate endpoints, one for retrieving each process result?

Or, a single endpoint which receives a command to execute, run the command, and return the output, putting the onus on the test to provide the appropriate command to run.

Reply:
I've implemented separate endpoints. I didn't want to make a generic endpoint that just executes passed in code. Even if it's just for tests, the thought of just executing unchecked remote code doesn't sound right.